source/developer/webhooks-incoming.rst

```diff
@@ -147,6 +147,8 @@ Troubleshooting
 
 To debug incoming webhooks in **System Console > Logs**, set **System Console > Logging > Enable Webhook Debugging** to ``true`` and set **System Console > Logging > Console Log Level** to ``DEBUG``.
 
+As of v5.14, When ``DEBUG`` Mode is ``TRUE``, enhanced logging is enabled and each webhook logged will have a ``request_id`` value in the log file to assist with matching issues to particular requests. 
```

Comment:
Was this for outgoing or incoming webhooks?

Reply:
This is absolutely for Outgoing webhooks, reverting.  I edited the wrong file.